examples/spotify-api-example/src/main/java/com/spotify/apollo/example/data/Track.java

```diff
@@ -0,0 +1,19 @@
+package com.spotify.apollo.example.data;
```

Comment:
I think we should use AutoValue for these classes, or else, at least make them immutable (final fields). AutoValue is much nicer (because you get equals() and toString() for free) if we want to write some tests for this example, which I think we should do.